src/Routing/UrlGenerator.php

```diff
@@ -141,6 +141,8 @@ private function prepareAlias($alias, array &$parameters)
         $hasAutoItem = false;
         $autoItems = $this->getAutoItems($parameters);
 
+        unset($parameters['auto_item']);
```

Comment:
Why did you change this? It has worked without it, hasn't it?

Reply:
No, the test failed after correctly implementing it 😉